.markdownlint.json

```diff
@@ -3,10 +3,12 @@
     "MD004": {"style": "asterisk"},
     "MD007": { "indent": 4 },
     "MD013": false,
+    "MD014": false,
     "MD024": { "siblings_only": true },
     "MD025": false,
     "MD033": false,
     "MD034": false,
+    "MD049": {"style": "underscore"},
```

Comment:
This one is for emphasis (italics), we have written them with `_` throughout so we ought to make it official.